src/OrchardCore.Modules/OrchardCore.ContentLocalization/OrchardCore.ContentLocalization.csproj

```diff
@@ -18,6 +18,8 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.DisplayManagement\OrchardCore.DisplayManagement.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Indexing.Abstractions\OrchardCore.Indexing.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Data.Abstractions\OrchardCore.Data.Abstractions.csproj" />
+    <ProjectReference Include="..\OrchardCore.Autoroute\OrchardCore.Autoroute.csproj" />
```

Comment:
Probably a bad idea to point to the Autoroute module directly. Might need to make an abstraction project for this one.

Reply:
I think content items can be retreived by Autoroute and by alias. So if you are looking at a page that it's url is "cloaked" with an Alias you won't be able to find it's contentItemId. I think you need to look for both cases.